app/src/androidTest/java/org/simple/clinic/drugs/PrescriptionRepositoryAndroidTest.kt

```diff
@@ -137,4 +136,16 @@ class PrescriptionRepositoryAndroidTest {
     assertThat(softDeletedPrescription.createdAt).isEqualTo(prescription.createdAt)
     assertThat(softDeletedPrescription.syncStatus).isEqualTo(SyncStatus.PENDING)
   }
+
+  @Test
+  fun facilities_should_be_overridable() {
```

Comment:
Copy Pasta. Shouldn't this be `prescriptions_should_be_overridable`?

Reply:
![2yrzbi4fzvhec1el28qxpau69ekfpp5ak7fwanxmafmfpyxpet7qdsnd87xdwo54lbbrincb2vusueiqeccjplrnefg8ukcsux2](https://user-images.githubusercontent.com/2387680/48477725-565be900-e828-11e8-9647-950a7ba9617d.png)

Reply:
Fixed